### DISCUSSION TITLE
无法使用磁盘模式引导

### DISCUSSION THREAD

**Discussion by mouyase** [kkkgo/PaoPaoGateWay#106] [问和答] [answered]
尝试将iso镜像写入U盘，或者硬盘，均无法引导系统，导致无法在物理机环境使用

虚拟机如果将iso挂载为硬盘而非光盘，也会无法引导，提示没有可引导的硬盘

**Comment by kkkgo** [comment]
iso文件直接引导会找不到init。下个版本会加入rufus写入支持。

**Reply by mouyase** [reply]
感谢大佬对小众需求的支持

**Comment by kkkgo** [comment, marked as answer]
最新版本已支持。https://github.com/kkkgo/PaoPaoGateWay/releases

**Reply by mouyase** [reply]
感谢付出
